frontend/public/components/cloud-services/catalog-source.tsx

```diff
@@ -135,6 +135,7 @@ export const CatalogSourceList = withFallback((props: CatalogSourceListProps) =>
 
   return props.loaded
     ? <React.Fragment>
+      <p className="co-m-pane__explanation" style={{marginTop: '-65px'}}>Catalogs are groups of Operators you can make available on the cluster. Subscribe and grant a namespace access to use the installed Operators.</p>
```

Comment:
This will cause the text to overlap the filter input at smaller widths.

@tlwu2013 Can we look at this as a follow on? The problem is that space is reserved for the filter bar. It's not a simple change unfortunately.

I'd like to find a general solution if we think we might want this on other pages, but I don't want to block this PR on that.

Reply:
@spadgett Removed the `margin-top` override.